flake8_black.py

```diff
@@ -52,17 +54,30 @@ def __init__(self, tree, filename="(none)", builtins=None):
         # see property self._file_mode for new black versions:
         self.file_mode = 0  # was: black.FileMode.AUTO_DETECT
 
-    def _load_black_config(self):
+    @property
+    @lru_cache()
```

Comment:
Why bother with the LRU? I've not profiled this, but perhaps flake8 ends up triggering this more than one?

Reply:
Just 4 times in an ordinary flake8 run over one file, so the impact is probably miniscule. I tend to add it to properties that are a little bit more complicated just for the peace of mind -- comes with the standard lib, shame not to use it.

Please tell me if you want it removed.